<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Macs-MacBook-Pro.local" failures="1" tests="2" name="com.anhtester.projects.cms.testcases.OrderTest" time="115.454" errors="0" timestamp="2023-11-02T10:47:28 IST" skipped="1">
  <testcase classname="com.anhtester.projects.cms.testcases.OrderTest" name="TC_OrderProduct" time="69.276">
    <failure message="Add to cart is failedBy.xpath: //h3[normalize-space()=&#039;Item added to your cart!&#039;]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Add to cart is failedBy.xpath: //h3[normalize-space()='Item added to your cart!']
at org.testng.Assert.fail(Assert.java:111)
at com.anhtester.keywords.WebUI.verifyElementVisible(WebUI.java:2068)
at com.anhtester.projects.cms.users.pages.order.OrderPage.order(OrderPage.java:73)
at com.anhtester.projects.cms.testcases.OrderTest.TC_OrderProduct(OrderTest.java:19)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- TC_OrderProduct -->
  <testcase classname="com.anhtester.projects.cms.testcases.OrderTest" name="TC_OrderProduct" time="46.178">
    <skipped/>
  </testcase> <!-- TC_OrderProduct -->
</testsuite> <!-- com.anhtester.projects.cms.testcases.OrderTest -->
